test/module/irohad/ametsuchi/postgres_executor_test.cpp

```diff
@@ -98,6 +98,18 @@ namespace iroha {
                         true)));
       }
 
+      /**
+       * Check that command result contains specific error code
+       * @param result to be checked
+       * @param expected_code to be in the result
+       */
+      void checkErrorCode(const CommandResult &result,
+                          CommandError::ErrorCodeType expected_code) {
+        auto error = err(result);
+        ASSERT_TRUE(error);
+        ASSERT_EQ(error->error.error_code, expected_code);
```

Comment:
I have put this assert into the function in order to make the code more concise. There can be confusion about place, where it was triggered, because each test calls the function only once, and Gtest tells, in which test it failed